packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/common.tests.js

```diff
@@ -67,6 +67,34 @@ QUnit.module('basics', {}, () => {
         assert.equal($element.find('.' + INPUT_CLASS).attr('inputmode'), 'decimal', 'inputmode is correct');
     });
 
+    QUnit.module('Tabindex', () => {
+        const testCases = [
+            { initial: false, updated: true, expectedInitial: -1, expectedUpdated: 0 },
+            { initial: true, updated: false, expectedInitial: 0, expectedUpdated: -1 }
+        ];
+
+        testCases.forEach(({ initial, updated, expectedInitial, expectedUpdated }) => {
+            QUnit.test(`should be ${expectedInitial} if focusStateEnabled = ${initial} on init (T1275314)`, function(assert) {
+                const element = $('#numberbox').dxNumberBox({ focusStateEnabled: initial });
+                const $input = element.find(`.${INPUT_CLASS}`);
+
+                assert.strictEqual($input.prop('tabindex'), expectedInitial);
+            });
+
+            QUnit.test(`should update from ${expectedInitial} to ${expectedUpdated} when focusStateEnabled changes to ${updated} (T1275314)`, function(assert) {
+                const element = $('#numberbox').dxNumberBox({ focusStateEnabled: initial });
```

Comment:
we use dollar prefix for jQuery elements
```suggestion
                const $element = $('#numberbox').dxNumberBox({ focusStateEnabled: initial });
```

Reply:
[4c5c514](https://github.com/DevExpress/DevExtreme/pull/29033/commits/4c5c5140ed24f5d5e4aff478409ac2e3471ff629)